packages/docusaurus-1.x/examples/basics/languages.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const languages = [
+  {
+    enabled: true,
+    name: 'English',
```

Comment:
Have kept this english for now, could actually be an empty array something like a comment stating change if required